README.md

```diff
@@ -135,6 +135,8 @@ Add the `PushNotifications_Administration` page to the project navigation. This
 
 At this point you can deploy your application to the Mendix cloud. If you are using a Free App, simply click the `Run` button.
 
+You should make sure that the `Encryption.EncryptionKey` constant has a value before you start the application. If you are using a Free App, you should set a default value for the constant in the model. Otherwise, you can configure the constant value in the Mendix Cloud Portal.
```

Comment:
Please clarify: 
Does "Otherwise" refer to: _when you're not using a Free App_, or to: _If the constant does not have a value_?